CHANGELOG.md

```diff
@@ -4,7 +4,11 @@
 Guide: https://keepachangelog.com/en/1.0.0/
 -->
 
-## Unreleased
+## 2.14.0-beta.1
+
+- [Core] Update dependencies.
+**MapboxCommon**: v24.14.0-beta.1
+**MapboxCoreSearch**: v2.14.0-beta.1
 
 - [SearchUI] Fix alert presentation on iPad.
```

Comment:
This one change was released in 2.13.3.
Could you please copy [these changelog updates](https://github.com/mapbox/mapbox-search-ios/blob/v2.13.3/CHANGELOG.md#2132) for 2.13.3 into the main branch?